app/src/main/java/com/example/news/model/CachedData.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.news.model
+
+import androidx.room.PrimaryKey
+
+data class CachedData(
```

Comment:
please create a separate model for the database

Reply:
fixed